doc/classes/@GlobalScope.xml

```diff
@@ -849,11 +849,11 @@
 				[codeblocks]
 				[gdscript]
 				var a = [1, 2, 3]
-				print("a", "b", a) # Prints ab[1, 2, 3]
+				print("a", "b", a) # Prints "ab[1, 2, 3]"
```

Comment:
Looking at this example specifically, I wonder if a policy that is to use quotes for strings but not for number types is the right one.

What gets printed is:
```
ab[1, 2, 3]
```
which doesn't differ much from the output of `print(42)` or `print("ab[1, 2, 3]")`.

```
var a = [1, 2, 3]
print("a", "b", a)
print("ab[1, 2, 3]")
print(42)
```
prints:
```
ab[1, 2, 3]
ab[1, 2, 3]
42
```

So maybe we should use quotes *all the time* and not just for strings?

Reply:
I'll leave that up for you or someone else to decide. Doing this is bikeshedding a bit and would be a whole lot more lines to change, but I do not mind doing it so long as it's consistent.